src/pages/SearchPage.js

```diff
@@ -0,0 +1,175 @@
+    }
+    loadData();
+  }
+
+  return (
+    <Wrapper>
+      <Header />
+      <StyledForm onSubmit={handleSubmit}>
+        <label htmlFor="search">
+          <ScreenReaderOnly>Suche</ScreenReaderOnly>
+        </label>
+        <input
+          id="search"
+          name="search"
+          placeholder="Star Wars, Shameless,..."
+          type="search"
+          required
+        />
+      </StyledForm>
+      {isLoading && <LoadingSpinner />}
+      {results.length !== 0 ? (
+        <StyledList>
+          {results
+            .filter(result => result.media_type !== 'person')
+            .map(
+              ({
+                id,
+                name,
+                title,
+                poster_path,
+                release_date,
+                first_air_date,
+                media_type,
+              }) => (
+                <StyledListItem key={id}>
+                  <Link to={name ? `/serie/${id}` : `/film/${id}`}>
+                    <Poster
+                      src={
+                        poster_path
+                          ? `https://image.tmdb.org/t/p/w300${poster_path}`
+                          : require('../assets/images/poster.png')
+                      }
+                      alt={name ? `${name}` : `${title}`}
+                    />
+                  </Link>
+                  <TextBox>
+                    <span>{name ? `${name}` : `${title}`}</span>
+                    <span>
+                      {release_date
+                        ? `${
+                            release_date
+                              ? release_date.substr(0, 4)
+                              : 'kein Release Datum vorhanden'
+                          }`
+                        : `${
+                            first_air_date
+                              ? first_air_date.substr(0, 4)
+                              : 'kein Release Datum vorhanden'
+                          }`}{' '}
+                      - {media_type === 'movie' ? 'Film' : 'Serie'}
+                    </span>
+                  </TextBox>
+                </StyledListItem>
+              )
+            )}
+        </StyledList>
+      ) : (
+        <InfoBox>
+          <span>🕵🏼‍♀️ 🕵🏼 🕵🏻‍♂️</span>
+          <span>Bitte gib ein Suchbegriff ein</span>
+          <span> oder ändere deine Suche </span>
+        </InfoBox>
+      )}
+      <Navigation />
+    </Wrapper>
+  );
+}
+const Wrapper = styled.div`
+  margin-top: 60px;
+  margin-bottom: 68px;
+`;
+
+const StyledForm = styled.form`
+  background-color: var(--color-black);
+  border-bottom: 1px solid var(--border-color);
+  padding: 15px;
+  position: sticky;
+  top: 59px;
+  input {
+    padding: 10px 8px;
+    width: 100%;
+    color: inherit;
+    border-radius: var(--border-radius);
+    border: transparent;
+    background-color: var(--color-dark-gray);
+  }
+`;
+
+const StyledList = styled.ul`
+  list-style: none;
+  padding: 0px 25px;
+  margin: 0px;
```

Comment:
If it's zero than it should be written without 'px'
```suggestion
  padding: 0 25px;
  margin: 0;
```